Library/Homebrew/dependable.rb

```diff
@@ -20,8 +20,6 @@ def run?
   end
 
   def required?
-    # FIXME: Should `required?` really imply `!build?`? And if so, why doesn't
-    #        any of `optional?` and `recommended?` equally imply `!build?`?
```

Comment:
Was added 1.5 years ago by someone who is no longer a maintainer. It's not causing any user issues so "fixing" it would increase the chance of bugs with no benefit.